web/sites/default/settings.amezmo.php

```diff
@@ -0,0 +1,88 @@
+<?php
+/**
+ * @file
+ * Platform.sh settings.
+ */
+
+// Configure the database.
+if (isset($_ENV['APP_HOSTNAME'])) {
+  // TODO: FIND A WAY TO GET DB_HOST AND DB_USER
+  $databases['default']['default'] = [
```

Comment:
We can generate APP_HOSTNAME as a Drupal specific configuration that would be autogenerated by Amezmo, so I agree with this.